charger/ocpp/cs_core.go

```diff
@@ -17,12 +18,17 @@ func (cs *CS) OnAuthorize(id string, request *core.AuthorizeRequest) (*core.Auth
 }
 
 func (cs *CS) OnBootNotification(id string, request *core.BootNotificationRequest) (*core.BootNotificationConfirmation, error) {
-	cp, err := cs.ChargepointByID(id)
-	if err != nil {
-		return nil, err
+	if cp, err := cs.ChargepointByID(id); err == nil {
+		return cp.OnBootNotification(request)
 	}
 
-	return cp.OnBootNotification(request)
+	res := &core.BootNotificationConfirmation{
+		CurrentTime: types.Now(),
+		Interval:    60,
```

Comment:
Das ist in dem Fall wahrscheinlich etwas lang.
hier auch `Timeout` verwenden?
Ist ja dann das Retry-Intervall des CP.

Reply:
Eigentlich sollte der PR keine Funktion ändern- das ist schon immer so. Können wir aber gerne separat anpassen.

Reply:
Vorher wurde aber nie mit `Pending` sondern immer mit Accepted (am CP) geantwortet.
Das ist ein entscheidender Unterschied in der Bedeutung von `Interval`.

Reply:
Die Pending Antwort ist jetzt komplett neu. Ich habs im master auf Timeout geändert (genauso wie accepted). Vorher gabs da gar nichts.

Reply:
`Accepted` sollte wie vorher (am LP) auf 60s bleiben, da sonst manche Boxen mit dem Hearbeat-Intervall < 60s Probleme haben.
Im Fall von `Pending` ist die Bedeutung von `Interval:` aber eben nicht "Hearbeat-Interval" sondern "Retry-Interval". Also etwas anderes.
Da ist `Timeout` sinnvoll.